guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,173 @@
+- Automatically open channels anytime the user has on-chain funds - [Auto pilot](https://bitcoin.design/guide/how-it-works/lightning-services/#on-chain-funding)
+- Automatic channel creation (using [Dunder LSP](https://github.com/hsjoberg/dunder-lsp)) during initial deposit of funds
+- Support for custom LSP
+
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-users-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-users.jpg"
+   alt-text = "Characters representing Blixt's users"
+   caption = "Illustrations: [OpenPeeps](https://www.openpeeps.com/) by Pablo Stanley"
+   width = 800
+   height = 450
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process@2x.jpg"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process-mobile.jpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-ux-process.jpg"
+   alt-text = "Diagram of stages in the Blixt UX process"
+   width = 800
+   height = 929
+%}
+
+Since Blixt is an open source wallet, we collected user research simply by going on their Discord channel and collecting user comments and questions.
+
+## We discovered that Blixt users were struggling to open lightning channels
+
+We created a lot of different user flows to find a solution for the best method to open a channel for a user new to lightning, and eventually settled on this one. We decided on this user flow as it educates users first and then takes them to setting up the channel itself. 
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow-mobile.png"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-user-flow-mobile@2x.png"
+   alt-text = "Diagram of the Blixt user flow, from initial onboarding through setting up channels, backing up the wallet, scanning a QR, and sending first payment."
+   width = 1600
+   height = 644
+   layout = "full-width"
+%}
+
+## One of the biggest challenges we faced was
+
+Explaining the concept of channel opening to people who are new to lightning. The concept of opening a channel to make a transaction is totally new to them. Next, some users might not be native English speakers, so using words and terms that communicate clearly to a larger group of people was essential.
+
+## After re-designing the interface we tested the new design by doing usability tests. Here’s the process we followed:
```

Comment:
```suggestion
## Usability tests

After re-designing the interface, we tested the new design by doing usability tests. Here’s the process we followed:
```

These long titles get super huge. Maybe better to use titles just to hint at what's happening, and use body text for the details?